docs/tasks.md

```diff
@@ -4,27 +4,24 @@ A list of reusable Cumulus tasks. [Add your own.](adding-a-task.md)
 
 ## Tasks
 
-### [@cumulus/discover-granules](https://github.com/cumulus-nasa/cumulus/tree/master/tasks/discover-granules)
-Discover Granules in FTP/HTTP/SFTP endpoints
+### @cumulus/discover-granules
+Discover Granules in FTP/HTTP/HTTPS/SFTP/S3 endpoints
 
-- Schemas: See this module's [schema definitions](https://github.com/cumulus-nasa/cumulus/tree/master/tasks/discover-granules/schemas).
```

Comment:
Why did you remove the schemas and links?

Reply:
Building the docs was failing because the repository URL was not defined for some of the tasks.  I added a conditional to the build script that says, "if there is not a repository URL, don't add things that depend on that repository URL".  Any tasks that are missing a schema or other link are missing it because they aren't defining the homepage or repository URL in their package.json.